x-pack/platform/plugins/shared/ml/public/application/components/ml_page/ml_page.tsx

```diff
@@ -52,14 +59,34 @@ const ML_APP_SELECTOR = '[data-test-subj="mlApp"]';
 
 export const MlPageControlsContext = createContext<{
   headerPortal: HtmlPortalNode;
+  leftHeaderPortal: HtmlPortalNode;
+  rightHeaderPortal: HtmlPortalNode;
   setHeaderActionMenu?: AppMountParameters['setHeaderActionMenu'];
   setIsHeaderMounted: (v: boolean) => void;
   isHeaderMounted: boolean;
+  isLeftSectionMounted: boolean;
+  setIsLeftSectionMounted: (v: boolean) => void;
+  isRightSectionMounted: boolean;
+  setIsRightSectionMounted: (v: boolean) => void;
+  headerRestrictWidth?: boolean | number;
+  setHeaderRestrictWidth: (v: number | undefined) => void;
+  wrapHeader: boolean;
```

Comment:
The behavior looks slightly different between the Anomaly Detection and DFA pages. Is that by design?

For the Anomaly Detection page, the buttons all wrap below the header text as soon as they don't all fit on one line with the header text:
<img width="1498" height="290" alt="Screenshot 2025-08-26 at 17 02 02" src="https://github.com/user-attachments/assets/5125b198-057f-4bb5-8ad9-80eec61caea4" />

Where in the DFA pages, they first start wrapping to the right of the header text:
<img width="1066" height="262" alt="Screenshot 2025-08-26 at 17 03 00" src="https://github.com/user-attachments/assets/004e0670-5f1c-413d-8004-57a0fb96838d" />

I don't think it's worth spending much more time on this, but just wondered why there was a difference?